src/net/slashie/expedition/ui/console/CharExpeditionDisplay.java

```diff
@@ -82,12 +79,15 @@ public int showTitleScreen() {
 		File saveDirectory = new File("savegame");
 		File[] saves = saveDirectory.listFiles(new GameFiles.SaveGameFilenameFilter() );
 		boolean loadDisabled = false;
+		if (saves != null)

```

Comment:
This is fixed by creating a "savegame" directory on distro, it was missed from git on the transition